{
  "@type": "NewsArticle",
  "identifier": "2022--04--22--en--myfeed--HackerNews--NewsArticle--31116545",
  "url": "https://news.ycombinator.com/item?id=31116545",
  "headline": "Python 3.11 Preview: Task and Exception Groups",
  "publisher": {
    "@type": "Organization",
    "name": "HackerNews",
    "url": "https://news.ycombinator.com",
    "logo": "https://hn.buzzing.cc/avatar.png"
  },
  "description": "",
  "keywords": [],
  "author": {
    "@type": "Person",
    "name": "psalminen",
    "url": "https://news.ycombinator.com/user?id=psalminen"
  },
  "discussionUrl": "https://news.ycombinator.com/item?id=31116545",
  "sameAs": "https://realpython.com/python311-exception-groups/#cool-new-features",
  "dateCreated": "2022-04-22T00:43:47.810Z",
  "datePublished": "2022-04-21T23:42:26.000Z",
  "dateModified": "2022-04-22T00:43:47.810Z",
  "interactionStatistic": [
    {
      "@type": "InteractionCounter",
      "interactionType": {
        "@type": "LikeAction"
      },
      "userInteractionCount": 2
    },
    {
      "@type": "InteractionCounter",
      "interactionType": {
        "@type": "CommentAction"
      },
      "userInteractionCount": 0
    }
  ],
  "headline_zh-Hans": "Python 3.11预览。任务和异常组",
  "headline_zh-Hant": "Python 3.11預覽。任務和異常組",
  "@context": [
    "https://schema.org",
    {
      "@vocab": "http://schema.org/",
      "@language": "en",
      "headline_zh-Hans": {
        "@id": "headline",
        "@language": "zh-Hans"
      },
      "headline_zh-Hant": {
        "@id": "headline",
        "@language": "zh-Hant"
      },
      "@version": 1.1
    }
  ]
}